core/server/master/src/main/java/alluxio/master/meta/checkconf/ServerConfigurationChecker.java

```diff
@@ -114,7 +114,10 @@ public synchronized ConfigCheckReport getConfigCheckReport() {
       // during report regeneration, the change will be reflected in the next call to
       // getConfigCheckReport.
       mConfigDirty = false;
+      LOG.debug("Regenerating config check report");
       regenerateReport();
+    } else {
+      LOG.debug("Skipping regenerating config check report");
```

Comment:
Do you want to keep these?

Reply:
I kept them intentionally, I think they are reasonable in the debug scope